### PR TITLE
Make example folder self-contained

### PR DESCRIPTION
**Pull request by orangain**
## Motivation

Currently, the instructions for running the example folder use the npm scripts that is defined in the parent folder. This makes it a bit difficult for a user to copy the contents in the example folder to their project and run it.

Instead, I would propose to create a `package.json` file in the example folder and define the npm scripts in it. I would also propose that the `schmealint` library be installed from the npm registry in the example folder.

This way, the user can simply copy the contents of the example folder to their project and run it. I believe this will make it easier for the user to start using the library.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,9 +28,6 @@
   "scripts": {
     "build": "tsc",
     "lint": "eslint . -f visualstudio --report-unused-disable-directives --ext .js,.ts",
-    "run-example": "node build/cli -c ./example/.schemalintrc",
-    "start-example-db": "docker run -d --name dvd-rental -p 54321:5432 kristiandupont/dvdrental-image",
-    "stop-example-db": "docker stop dvd-rental",
```
Comment (orangain) [inline package.json:33]: These scripts have been moved to `example/package.json`. If you are using the scripts to run the example against a local version of the library, I would revert this change.

File: example/package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "scripts": {
+    "start-example-db": "docker run -d --name dvd-rental -p 54321:5432 kristiandupont/dvdrental-image",
+    "stop-example-db": "docker stop dvd-rental",
+    "lint:schema": "schemalint"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "schemalint": "^1.1.0"
```
Comment (kristiandupont) [inline example/package.json:15]: Maybe this should just be version `"*"`?

**Comment by kristiandupont** [comment]
Let me know if you agree with my comment, if not I am happy to just merge this.

File: example/package.json (hunk 2 above)
Comment (orangain) [reply example/package.json]: I agree with you. Using `"*"` would save us the trouble of keeping up with the latest version. Update in https://github.com/kristiandupont/schemalint/pull/340/commits/954dd4817a2743f85517766cae1432674a2118dc.

**Comment by kristiandupont** [comment]
🙏
